site/content/en/docs/tasks/run_plain_pods.md

```diff
@@ -0,0 +1,90 @@
+---
+title: "Run A Plain Pod"
+date: 2023-09-27
+weight: 6
+description: >
+  Run a Kueue scheduled Pod.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running plain Pods.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+1. By default, the integration for `v1/pod` is not enabled.
+   Learn how to [install Kueue with a custom manager configuration](/docs/installation/#install-a-custom-configured-released-version)
+   and enable the `pod` integration.
+
+   A configuration for Kueue with enabled pod integration would look like follows:
+   ```yaml
+   apiVersion: config.kueue.x-k8s.io/v1beta1
+   kind: Configuration
+   integrations:
+     frameworks:
+      - "pod"
+     podOptions:
+       # You can change namespaceSelector to define in which 
+       # namespaces kueue will manage the pods.
+       namespaceSelector:
+         matchExpressions:
+         - key: kubernetes.io/metadata.name
+           operator: NotIn
+           values: [ kube-system, kueue-system ]
+       # Kueue uses podSelector to manage pods with particular 
+       # labels. The default podSelector will match all the pods. 
+       podSelector:
+         matchExpressions:
+         - key: kueue-job
+           operator: In
+           values: [ "true", "True", "yes" ]
+   ```
+
+2. Pods that belong to other API resources managed by Kueue are excluded from being queued by `pod` integration. 
+   For example, pods managed by `batch/v1/Job` won't be managed by `pod` integration if the `job` integration is enabled.
```

Comment:
```suggestion
   For example, pods managed by `batch/v1.Job` won't be managed by `pod` integration if the `job` integration is enabled.
```

Is it only if enabled?

Reply:
> Is it only if enabled?

Oh, my bad. It has to be registered but not necessarily enabled.